### DISCUSSION TITLE
Exception during StreamSystem.Create() not exposed by library

### DISCUSSION THREAD

**Discussion by plewam** [rabbitmq/rabbitmq-stream-dotnet-client#354] [Q&A]
Hi All,

I use the most recent 1.8.0-rc2 version together with rmq 3.12.12.
During `StreamSystem.Create()` I see the below logging entry.

Inside the logging entry there is a `SocketException`, and in fact the stream system is not working. Any help would be appreciated.

`2024-02-07 14:18:56.830603 [DBG] [051_010] Client connected to IPEndPoint { AddressFamily: InterNetwork, Address: IPAddress { AddressFamily: InterNetwork, ScopeId: "The property accessor threw an exception: SocketException", IsIPv6Multicast: False, IsIPv6LinkLocal: False, IsIPv6SiteLocal: False, IsIPv6Teredo: False, IsIPv6UniqueLocal: False, IsIPv4MappedToIPv6: False, Address: 16777343 }, Port: 5552 } `

**Comment by plewam** [comment]
Update: The issue does not occur with rmq 3.13.0-rc.4

**Reply by Gsantomaggio** [reply]
@plewam Thank you, but It is not totally clear the bug to me. 

I'd like to reproduce the error even if it does not occur `3.13.0-rc.4.` 

The log you see:
https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/blob/dd8a4ce678b5e86dbf8bc6cb3b314f217c9b6584/RabbitMQ.Stream.Client/StreamSystem.cs#L92-L101


So, it means that the connection was created correctly else it would raise an error:
https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/blob/dd8a4ce678b5e86dbf8bc6cb3b314f217c9b6584/RabbitMQ.Stream.Client/StreamSystem.cs#L103-L118


Do you have a way to reproduce it with `3.12.12.`
